tests/test_dict_encoding.py

```diff
@@ -11,8 +11,8 @@
 import numpy as np
 import tskit
 
-import _msprime as c_module
```

Comment:
Much smaller change is to have ``import msprime._mspriem as c_module``

Reply:
Yes, but why a different convention in this module? Is it for readability?

Reply:
It's because we've copied it directly from the tskit source code, and the idea was to be able to update it over time by copying the new code in and not having to replace loads of ``_tskit`` with ``_msprime``'s. Also we can diff the two versions of the file more easily then to see if they're in sync.

Reply:
Ah, of course, makes sense.